vault.py

```diff
@@ -1467,17 +1512,18 @@ def get_approver(ctx, path):
         return None
 
 
-def get_doi(ctx, path):
+def get_doi(ctx, path, doi):
```

Comment:
Safer to add a default to doi: `doi="version"` and add a check to set it to version if it is not `base`?

```
def get_doi(ctx, path, doi="version"):
    if doi != "base":
        doi="version"
```